plugin.rb

```diff
@@ -737,6 +737,16 @@ class ::ListController
     end
   end
 
+  on(:post_moved) do |post, original_topic_id|
+    assignment = Assignment.where(topic_id: original_topic_id, target_type: "Post", target_id: post.id).first
+    return if !assignment
```

Comment:
```suggestion
    next if !assignment
```

Return in a block is probably not what you want here, it returns from the higher scope.